### PR TITLE
feat(external-api): add local subject command

### PR DESCRIPTION
**Pull request by mihhu**
Extended the external API to support setting a local subject from all participants (regardless of the role).

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1045,6 +1045,9 @@ var config = {
     // Sets the conference subject
     // subject: 'Conference Subject',
 
+    // Sets the conference local subject
+    // localSubject: 'Conference Local Subject',
```
Comment (quitrk) [inline config.js:1049]: this should also be added to configWhitelist.js

File: react/features/base/conference/functions.js [hunk 2]
```diff
@@ -180,9 +180,9 @@ export function getConferenceName(stateful: Function | Object): string {
     const state = toState(stateful);
     const { callee } = state['features/base/jwt'];
     const { callDisplayName } = state['features/base/config'];
-    const { pendingSubjectChange, room, subject } = getConferenceState(state);
+    const { localSubject, room, subject } = getConferenceState(state);
```
Comment (quitrk) [inline react/features/base/conference/functions.js:183]: shouldn't we still use `pendingSubjectChange`?

Comment (mihhu) [reply react/features/base/conference/functions.js:183]: `pendingSubjectChange` is only used to update the global `subject` now (when the participant has a moderator role) and for local purposes we provide this new command instead.

**Comment by mihhu** [comment]
Please Jenkins test this please.

**Comment by mihhu** [comment]
Please Jenkins test this please.

**Comment by mihhu** [comment]
Please Jenkins test this please.

**Comment by mihhu** [comment]
Please Jenkins test this please.

**Review by quitrk — APPROVED** [review]

**Pull request merged**